fedimint-server/src/config/api.rs

```diff
@@ -573,12 +574,12 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConfigGenApi>> {
         },
         api_endpoint! {
             "status",
-            async |config: &ConfigGenApi, context, _v: ()| -> ServerStatus {
-                if context.has_auth() {
-                    Ok(config.status().await)
-                } else {
-                    Err(ApiError::unauthorized())
-                }
+            async |config: &ConfigGenApi, _context, _v: ()| -> StatusResponse {
+                let server = config.server_status().await;
+                Ok(StatusResponse {
+                    server,
+                    consensus: None
+                })
```

Comment:
I think we'll want to reconsider making this a public endpoint eventually, but ok for now.